musonant/questioner
# Refactor API

#### What does this PR do?
- Refactor API
#### Description of Task to be completed?
-  Create middleware for handling parameter
- Create file response.js to handle various kinds of api responses
- Refactor controllers to use responses from response.js
#### How should this be manually tested?
- Clone this repository
- Run `npm install` followed by `npm run test`
- Previous tests should still be passed
#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/163059328
#### Screenshots (if appropriate)
NA
#### Questions:`
NA
